cross/ezstream/PLIST

```diff
@@ -0,0 +1,13 @@
+bin:bin/ezstream
+rsc:bin/ezstream-file.sh
+shr:share/examples/ezstream/ezstream-file_template.xml
+shr:share/examples/ezstream/ezstream_metadata.xml
```

Comment:
afaik we don't have `shr` type that should be `rsc`

Reply:
Ok noted, I will replace sur by rsc